src/app/RenderTable.tsx

```diff
@@ -0,0 +1,100 @@
+import { SdkResult } from "alphadex-sdk-js/lib/models/sdk-result";
+import { useContext } from "react";
+import { AdexStateContext } from "./contexts";
+
+//TODO:
+//1. Add datafield for executed price when available from Adex
+//2. onClick Cancel Order function
+//3. onClick Change pair when pair on the table is clicked
+//4. Display "Connect Wallet to see History" if no wallet is connected
+//5. Display "No Orders" if Wallet is connected but no History
+
+interface RenderTableProps {
+  response: SdkResult | null;
+  selectedTable: string | null;
+}
+
+function formatDate(date: string) {
+  return new Date(date)
+    .toLocaleString("en-US", {
+      year: "numeric",
+      month: "2-digit",
+      day: "2-digit",
+      hour: "2-digit",
+      minute: "2-digit",
+      second: "2-digit",
+      hour12: false,
+    })
+    .replace(/(\d+)\/(\d+)\/(\d+), (\d+:\d+:\d+)/, "$3-$1-$2 $4");
+}
+
+function filterOrdersByStatus(orders: any[], status: string) {
+  return orders.filter((order) => order.status === status);
+}
+
+export function RenderTable({ response, selectedTable }: RenderTableProps) {
+  const adexState = useContext(AdexStateContext);
+  let orders = response ? response.data.orders : [];
+
+  switch (selectedTable) {
+    case "OpenOrders":
+      orders = filterOrdersByStatus(orders, "PENDING");
+      break;
+    case "TradeHistory":
+      orders = filterOrdersByStatus(orders, "COMPLETED");
+      break;
+    default:
+      break;
+  }
+
+  return (
+    <div>
+      {selectedTable ? (
+        <table>
+          <thead>
+            <tr>
+              <th>Pair</th>
+              <th>ID</th>
```

Comment:
What are these columns? I think we need only human readable Pair Name, no need for pair ID, if it's order id it should be later in the columns
please refer to figma to get the correct columns seprarately for all 3 tables: - https://www.figma.com/file/P7pfzKwJ4G6ClapXfl61D3/Dexter-UI?type=design&node-id=352-20276&mode=design&t=WbFm5oHBrdWQTz3h-0

Reply:
`<th>Pair</th>` would display the pair name ie ADEX / XRD etc
`<th>ID</th>` while i dont think this is relevant information for the average user, i just included it to be displayed for now just to display, but i can remove that so the average user wouldnt fixate on what that is.

Will update this to be closer to the figma display.

Reply:
I agree that any of the id fields are probably not relevant for end-users so would agree that we remove all ids from the ui. I do understand it is very useful when developing though.